src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/01-windows-installation.md

```diff
@@ -0,0 +1,194 @@
+```
+
+![msys2](/splashkit/windows-installation-fig10.png)
+
+### Execute skm
+
+**Restart** your terminal, then execute the following:
+
+```bash
+skm
+```
+
+Then the command:
+
+```bash
+skm global install
+```
+
+![msys2](/splashkit/windows-installation-fig11.png)
+
+## Install Visual Studio Code
+
+### Download Visual Studio Code
+
+To install Visual Studio Code, go to the [Visual Studio Code website](https://code.visualstudio.com/) and download the `.exe` file.
+
+![msys2](/splashkit/windows-installation-fig12.png)
+
+### Installing Necessary Extensions
+
+To install the necessary extensions, keep the terminal open and run the following commands:
+
+#### C# Extension
+
+[C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp):
+
+```bash
+code --install-extension ms-dotnettools.csharp
+```
+
+![Terminal](/splashkit/linux-installation-fig12.png)
+
+#### C# Dev Kit
+
+[C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit):
+
+```bash
+code --install-extension ms-dotnettools.csdevkit
+```
+
+![Terminal](/splashkit/linux-installation-fig13.png)
+
+#### Intellicode for C# Dev Kit
+
+[Intellicode for C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscodeintellicode-csharp):
+
+```bash
+code --install-extension ms-dotnettools.vscodeintellicode-csharp
+```
+
+![Terminal](/splashkit/linux-installation-fig14.png)
+
+#### C++ Extension Pack
+
+[C++ Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-extension-pack):
+
+```bash
+code --install-extension ms-vscode.cpptools-extension-pack
+```
+
+## Installing Necessary Extensions for Visual Studio Code
+
+To install the necessary extensions, keep the terminal open and run the following commands:
+
+### C# Extension
+
+[C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp):
+
+```bash
+code --install-extension ms-dotnettools.csharp
+```
+
+![Terminal](/splashkit/windows-installation-fig15.png)
+
+### C# Dev Kit
+
+[C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit):
+
+```bash
+code --install-extension ms-dotnettools.csdevkit
+```
+
+![Terminal](/splashkit/windows-installation-fig16.png)
+
+### Intellicode for C# Dev Kit
+
+[Intellicode for C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscodeintellicode-csharp):
+
+```bash
+code --install-extension ms-dotnettools.vscodeintellicode-csharp
+```
+
+![Terminal](/splashkit/windows-installation-fig17.png)
+
+### C++ Extension Pack
+
+[C++ Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-extension-pack):
+
+```bash
+code --install-extension ms-vscode.cpptools-extension-pack
+```
+
+![Terminal](/splashkit/windows-installation-fig18.png)
+
+### Installing .net SDK
```

Comment:
Guides up until this point have been detailed/verbose. The final 3 (.NET SDK, Docker, Python) are just links to external pages. Recommend having these at the same level of detail for consistency.

Reply:
Resolved without comment? Please address the comment before resolving.